modules/living_spaces_event/config/install/search_api.index.event_invites_and_chat_appointments.yml

```diff
@@ -0,0 +1,181 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.dip_chat_appointment.field_appointmentdate
+    - field.storage.dip_chat_appointment.field_counselor
+    - field.storage.dip_chat_appointment.field_moderators
+    - field.storage.living_spaces_event.field_end_date
+    - field.storage.living_spaces_event.field_start_date
+    - search_api.server.living_spaces
+  module:
+    - search_api_solr
+    - dip_chat_counseling
+    - living_spaces_event
+    - taxonomy
+    - search_api
+third_party_settings:
+  search_api_solr:
+    finalize: false
+    commit_before_finalize: false
+    commit_after_finalize: false
+    debug_finalize: false
+    highlighter:
+      maxAnalyzedChars: 51200
+      fragmenter: gap
+      usePhraseHighlighter: true
+      highlightMultiTerm: true
+      preserveMulti: false
+      regex:
+        slop: 0.5
+        pattern: blank
+        maxAnalyzedChars: 10000
+      highlight:
+        mergeContiguous: false
+        requireFieldMatch: false
+        snippets: 3
+        fragsize: 0
+    mlt:
+      mintf: 1
+      mindf: 1
+      maxdf: 0
+      maxdfpct: 0
+      minwl: 0
+      maxwl: 0
+      maxqt: 100
+      maxntp: 2000
+      boost: false
+      interestingTerms: none
+    term_modifiers:
+      slop: 3
+      fuzzy: 1
+    advanced:
+      index_prefix: ''
+      collection: ''
+      timezone: ''
+    multilingual:
+      limit_to_content_language: false
+      include_language_independent: true
+id: event_invites_and_chat_appointments
+name: 'Event invites and Chat appointments'
+description: ''
+read_only: false
+field_settings:
+  field_appointmentdate:
+    label: 'Appointment date'
+    datasource_id: 'entity:dip_chat_appointment'
+    property_path: field_appointmentdate
+    type: date
+    dependencies:
+      config:
+        - field.storage.dip_chat_appointment.field_appointmentdate
+  field_counselor:
```

Comment:
there is no field_counselor, field_moderators, etc fields in living spaces profile. they should be removed from that configs and should be added via rewrite in dip_chat_counseling module